migrate-dao.js

```diff
@@ -1,5 +1,6 @@
 const utils = require('./utils.js')
 async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logTx, previousMigration: { base } }) {
+  opts.gas = 6721975
```

Comment:
what is this gas number ?

Reply:
The max gas limit of ganache

Reply:
How come it is greater than ARC's ganache gas limit ?

Reply:
I guess this is the default of Ganache...

Reply:
though the script is not just for ganache.(and I am not sure it is the default).